resolver/src/resolver.rs

```diff
@@ -191,7 +191,7 @@ mod tests {
 
     #[test]
     #[ignore]
-    #[cfg(unix)]
+    #[cfg(not(all(target_os = "windows", target_pointer_width = "32")))]
```

Comment:
similar comment to above... though since it's a test case, maybe something else?